Makefile

```diff
@@ -0,0 +1,26 @@
+TAG ?= master
+
+build:
+	go build ./...
+
+.PHONY: test
+test:
+	bash test/scripts/unittests.sh
+
+hello:
+	curl localhost:8080/api/v1/resource/my-resource
+
+mlmd-proto:
+	rm -rf /tmp/ml-metadata && \
+	git clone --branch $(TAG) https://github.com/google/ml-metadata.git /tmp/ml-metadata && \
+	rm -rf ml_metadata/proto && \
+	mkdir ml_metadata/proto && \
+	cp /tmp/ml-metadata/ml_metadata/proto/*.proto ml_metadata/proto && \
+	protoc -I . ml_metadata/proto/*.proto --go_out=plugins=grpc:. && \
```

Comment:
Let's not use developer-installed protoc and building from ml_metadata head. Instead, let's use Bazel for proto generation instead. This ensures the build is hermetic and we won't see proto files oscillating due to dev setup (this was a problem in pipelines). See the example I did using the script + BUILD rule for how to do this. Alternately, if you're not sure how to do it, leave a TODO here for me and I'll change this part later.

Reply:
SG. I added a TODO